en/micro-integrator/docs/develop/create-integration-project.md

```diff
@@ -0,0 +1,117 @@
+# Creating an Integration Project
+
+An integration project consists of one or several project directories. These directories store the various artifacts that you create for your integration sequence. An integration project can be created as a Maven Multi Module (MMM) project by default. This enables you to add ESB Configs, Composite Exporter, Registry Resources, Connector Exporter, Docker Exporter, and Kubernetes Exporter as sub-modules to the project. 
+
+An integration project is the recommended way of creating an “Integration Solution” for the WSO2 Enterprise Integrator as it simplifies the CICD workflow.
+
+**To create an integration project:**
+
+1. [Download](https://wso2.com/integration/integration-studio/) and [install WSO2 Integration Studio](installing-WSO2-Integration-Studio/).
+
+2. Open WSO2 Integration Studio and click **New Integration Project** in the **Getting Started** view as shown below.
+    <img src="../../assets/img/new-project/new-integration-project.png" title="New Integration Project" width="700" alt="New Integration Project"/>
+
+3. In the **New Integration Project** dialog that opens, enter a name for your integration project. Select the relevant check boxes if you want to create **Registry Resources**, **Connector Exporter**, **Docker Exporter**, or **Kubernetes Exporter** in addition to the **ESB Configs** and **Composite Exporter**.
+    <img src="../../assets/img/new-project/create-a-project.png" title="Create a New Integration Project" width="700" alt="Create a New Integration Project"/>
+
+## Sub projects
```

Comment:
Sub projects or 'modules'?

Reply:
The word "modules" is already used for something else, so we decided to use projects here to avoid confusion.